ui/etc/epics-install.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
```

Comment:
This directory should get moved up one level. It has nothing to do with the ui.

Reply:
5a20ae7